src/main/openapi/rems.yaml

```diff
@@ -624,6 +811,125 @@ components:
           type: array
           items:
             type: object
+    V2License:
+      required:
+        - license/type
+        - license/title
+        - license/id
+        - license/enabled
+        - license/archived
+      properties:
+        license/type:
+          type: string
+          enum:
+            - link
+            - attachment
+            - text
+        license/title:
+          $ref: '#/components/schemas/LocalizedString'
+        license/id:
+          type: integer
+          format: int64
+        license/enabled:
+          type: boolean
+        license/archived:
+          type: boolean
+    Form:
+      required:
+        - form/id
+        - form/internal-name
+        - form/external-title
+        - form/fields
+      properties:
+        form/id:
+          type: integer
+          format: int64
+        form/internal-name:
+          type: string
+        form/external-title:
+          $ref: '#/components/schemas/LocalizedString'
+        form/fields:
+          type: array
+          items:
+            $ref: '#/components/schemas/Field'
+    Field:
+      required:
+        - field/private
+        - field/title
+        - field/visible
+        - field/type
+        - field/value
+        - field/id
+        - field/optional
+      properties:
+        field/private:
+          type: boolean
+        field/title:
+          $ref: '#/components/schemas/LocalizedString'
+        field/visible:
+          type: boolean
+        field/type:
+          type: string
+          enum:
+            - description
+            - email
+            - date
+            - phone-number
+            - table
+            - header
+            - texta
+            - option
+            - label
+            - multiselect
+            - ip-address
+            - attachment
+            - text
+        field/value:
+          example: value
+          description: 'A string for most fields, or [[{"column": string, "value": string}]] for table fields'
+          x-oneOf:
```

Comment:
Regarding the use of x-oneOf, I have attempted to replace it with oneOf, but it didn't work. I spent a few minutes on it without finding a proper solution.

It seems that oneOf does not work well when choosing between a primitive type and array.

I don't want to spend more time on this right now, maybe it could be part of another ticket?

Reply:
Sure, no problem. Hard code it to type string, as we need only to support  attachment type, and then we see if we need to support the other type later